packages/cli/src/commands/translate/sync-keys/utils.ts

```diff
@@ -137,7 +137,9 @@ export const parseLocaleFiles = (filePath: string): ParsedTuple => {
               // eslint-disable-next-line @silverhand/fp/no-mutation
               nestedObject[key] = [value, false];
             } else {
-              consoleLog.fatal('Unsupported comment:', comment);
+              // eslint-disable-next-line @silverhand/fp/no-mutation
+              nestedObject[key] = [value, true];
+              consoleLog.warn('Unsupported comment:', comment);
```

Comment:
Not enough context for this. What is this for? A existing bug?

Reply:
no, because i wanted to make sure there's no unexpected comment in the phrases. fatal was too strict and cannot work with `/** @deprecated */` in this pr